### PR TITLE
Add possibility to skip configure and install steps

### PR DESCRIPTION
**Pull request by Blackhex**
Adds possibility to skip either or both the configure and install steps using corresponding environment variables.

### CODE REVIEW

File: .github/scripts/build-binutils.sh [hunk 1]
```diff
@@ -6,6 +6,8 @@ TARGET=${TARGET:-aarch64-w64-mingw32}
 BUILD_PATH=${BUILD_PATH:-$PWD/build-$TARGET}
 BUILD_MAKE_OPTIONS=-j$(nproc)
 INSTALL_PATH=${INSTALL_PATH:-~/cross}
+RUN_CONFIG=${RUN_CONFIG:-1}
```
Comment (eukarpov) [inline .github/scripts/build-binutils.sh:9]: Why not zero?

File: .github/scripts/build-binutils.sh [hunk 2]
```diff
@@ -15,20 +17,25 @@ set -x # echo on
 mkdir -p $BUILD_PATH/binutils
 cd $BUILD_PATH/binutils
 
-echo "::group::Configure binutils"
-../../code/$BINUTILS_VERSION/configure \
-    --prefix=$INSTALL_PATH \
-    --target=$TARGET
-echo "::endgroup::"
+if [ $RUN_CONFIG = 1 ] ; then
```
Comment (eukarpov) [inline .github/scripts/build-binutils.sh:20]: if config has never been run, maybe it makes sense to run it anyway or stop execution

File: .github/scripts/build-binutils.sh (hunk 1 above)
Comment (Blackhex) [reply .github/scripts/build-binutils.sh:9]: Please elaborate your thoughts. I'd expect that one would expect full clean build by default.

File: .github/scripts/build-binutils.sh (hunk 2 above)
Comment (Blackhex) [reply .github/scripts/build-binutils.sh:20]: That's true. So far I left that decision to the user. Do you think it's worth to add detection whether the build was already configured here?

File: .github/scripts/build-binutils.sh (hunk 1 above)
Comment (eukarpov) [reply .github/scripts/build-binutils.sh:9]: please nvm, misreading as -1 )

File: .github/scripts/build-binutils.sh (hunk 2 above)
Comment (eukarpov) [reply .github/scripts/build-binutils.sh:8]: I think it is worth to add at least a warning message and stop execution.

**Comment by eukarpov** [comment]
> Code has significantly changed, please (re-)review.

Looks got for me. I have only the suggestion regarding skipping configuration.

**Comment by Blackhex** [comment]
Migrated to https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/4

**Pull request closed without merging**